schemas/events/v1/events.json

```diff
@@ -55,6 +55,22 @@
       "type": "string",
       "pattern": "https://console.redhat.com/api/schemas/.*"
     },
+    "title": {
+      "description": "Human readable text to identify the event",
```

Comment:
Should we try to make the description more clear that this applies to the entire payload no matter how many individual events are in it?